contrib/yolov5/setup.py

```diff
@@ -0,0 +1,22 @@
+from setuptools import find_namespace_packages, setup
+
+setup(
+    name="rikai-yolov5",
+    version="0.0.1",
+    license="Apache License, Version 2.0",
+    author="Rikai authors",
+    author_email="rikai-dev@eto.ai",
+    url="https://github.com/eto-ai/rikai",
+    python_requires=">=3.7",
+    install_requires=["rikai", "yolov5==5.0.8"],
```

Comment:
so do we want to maintain a new version of this `contrib` package for each yolov5 new version>

Reply:
Updated, restrict it in [5.0.0, 6.0.0), because I have not tested it for 6.x.